packages/fxa-auth-server/test/local/senders/mjml-emails.ts

```diff
@@ -41,7 +41,7 @@ const productMetadata = {
   'product:privacyNoticeDownloadURL': SUBSCRIPTION_PRIVACY_URL,
 };
 
-const MESSAGE = {
+const locals = {
```

Comment:
This was added to check that the image alt texts for `productNameNew` and `productNameOld` were included in the `subscriptionUpgrade`/`subscriptionDowngrade` emails.

Reply:
Thanks for the catch! Is there a reason we needed to change this from `MESSAGE` to `locals`, do you mean it wouldn't work without changing it or did this just seem like a more appropriate name? I thought `locals` was a keyword in EJS, the name change seems nice but I think we'd want to propagate it further by changing all `MESSAGE` references to `locals` instead of creating duplicate variable like `const MESSAGE = locals`, but it's possible I'm just missing something here 🙂

Reply:
It did not seem weird until you put it that way. An update is being made. Thanks!